components/menu/stories/menu.stories.js

```diff
@@ -211,9 +206,9 @@ TraySubmenu.argTypes = {
 };
 TraySubmenu.args = {
 	selectionMode: "multiple",
-	customStyles: {
-		"--mod-menu-inline-size": "100%",
-	},
+	// customStyles: {
```

Comment:
Should we just delete this `customStyles` object now?

Reply:
Good point, I've updated this